packages/graphql-serve/README.md

```diff
@@ -30,354 +42,80 @@ yarn add graphql-serve
-
-Remove all data from the database.
-
-```js
-// server: TestxServer
-await server.start();
-
-// some mutations
+Options:
+  --version    Show version number                                     [boolean]
+  --model, -m  Specify the models directory                             [string]
+  --port, -p   Specify the port on which to listen on                   [number]
+  -h, --help   Show help                                               [boolean]
 
-await server.cleanDatabase();
+Examples:
+  gqlserve --model=model  generate schema from models in the model directory
+  gqlserve --port=8000    server listens on port 8000
 ```
 
-It can be useful to reset the server state to a known point before each tests.
-
-#### `setData(data: ImportData): Promise<void>`
-
-Clean the database and insert the passed data directly to the database without
-passing through GraphQL mutations.
-
-```js
-// server: TestxServer
-await server.bootstrap();
-await server.setData({
-  item: [
-    { id: 1, name: "one" },
-    { id: 2, name: "two" }
-  ]
-});
-```
-
-It can be useful to reset the server to an initial state before each tests
-without triggering subscriptions, or data business logics.
-
-If you don't know the database structure you can use the `getDatabaseSchema()`
-method.
-
-#### `getGraphQlSchema(): Promise<string>`
-
-Return the generated GraphQL schema.
-
-```js
-// server: TestxServer
-await server.bootstrap();
-console.log(await server.getGraphQlSchema());
-```
-
-```graphql
-type Item {
-  id: ID!
-  name: String!
-}
-
-input ItemInput {
-  name: String!
-}
-
-input ItemFilter {
-  id: ID
-  name: String
-}
-
-type Query {
-  findItems(fields: ItemFilter!): [Item!]!
-  findAllItems: [Item!]!
-}
-
-type Mutation {
-  createItem(input: ItemInput!): Item!
-  updateItem(id: ID!, input: ItemInput!): Item!
-  deleteItem(id: ID!): ID!
-}
-
-type Subscription {
-  newItem: Item!
-  updatedItem: Item!
-  deletedItem: ID!
-}
-```
-
-#### `getDatabaseSchema(): Promise<{[table: string]: string[]}>`
-
-Return the generated database schema.
+Under to hood we use Graphback to parse the Type Definitions/Data Model and
+generate the GraphQL schema and resolvers. See the
+[Graphback Docs on Data Model Definition](https://graphback.dev/docs/datamodel)
 
-```js
-// server: TestxServer
-await server.bootstrap();
-console.log(await server.getDatabaseSchema());
-```
 
-```js
-{
-  item: ["id", "name", "created_at", "updated_at"];
-}
-```
+### printSchema
 
-#### `getQueries(): Promise<{[query: string]: string}>`
+The printSchema command prints what the generated GraphQL Schema of the server would look like.
 
-Return ready-to-use client queries.
+The `--model` argument works the same way here:
 
-```js
-// server: TestxServer
-await server.bootstrap();
-console.log((await server.getQueries()).findAllItems);
 ```
-
-```graphql
-query findAllItems {
-  findAllItems {
-    ...ItemFields
-  }
-}
-
-fragment ItemFields on Item {
-  id
-  name
-}
+$ printSchema --model=models
```

Comment:
I to think that we need to have this command as subcommand of serve

Reply:
I will make another PR for that